tee-worker/omni-executor/rpc-server/src/verify_auth.rs

```diff
@@ -100,9 +104,11 @@ pub fn verify_auth_token_authentication(
 	ctx: Arc<RpcContext>,
 	sender: &Identity,
```

Comment:
What do you thing about getting rid of inner sender to omni_account conversion and require `omni_account: String` explicitly ?

Reply:
I've updated the code 👍🏼